public/app/plugins/datasource/influxdb/components/VisualInfluxQLEditor/Seg.tsx

```diff
@@ -101,7 +101,7 @@ const SelSingleLoad = ({ loadOptions, allowCustomValue, onChange, onClose }: Sel
   const [loadState, doLoad] = useAsyncFn(loadOptions, [loadOptions]);
 
   useEffect(() => {
-    doLoad();
+    doLoad('');
```

Comment:
Just curious, why is the input to `doLoad()` an empty string?

Reply:
it's a strange corner-case. the `doLoad` function's type is: a function that takes a string-param.
so i must call it with a string-param.

now, in this specific case i do know that the param is ignored in the function, i could have called it with any-string param. i chose empty-string.

(there probably exists a nicer and longer fix to this problem, but i did not want to make the diff for this PR bigger than necessary)